tests/include/types/resourcesTest.php

```diff
@@ -43,12 +44,14 @@ public function testResource()
         $instance->resource = array('resource');
         $this->assertFalse($instance->validate());
         $this->assertContains('resources.resource', $instance->getError());
-        $resource = array(new resource('uri', null, 1, 'localURI'));
+        $resource = array(new resource('uri', null, 1, 'localURI','2016-03-11T14:23:23+00:00'));
         $instance->resource = $resource;
+        $instance->lastModifiedDate = '2016-03-11T14:23:23+00:00';
```

Comment:
please set the lastModifiedDate in the resources constructor like this ```new resources(NULL, NULL, '2016-03-11T14:23:23+00:00')```

it makes the test easier to understand